internal/utils/hash.go

```
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"crypto/sha256"
+
+	"golang.org/x/crypto/ripemd160"
```

golangci-lint (1.64.8)

SA1019: "golang.org/x/crypto/ripemd160" is deprecated: RIPEMD-160 is a legacy hash and should not be used for new applications. Also, this package does not and will not provide an optimized implementation. Instead, use a modern hash like SHA-256 (from crypto/sha256). 

(staticcheck)